spec/features/recipes/show_recipe_page_spec.rb

```diff
@@ -0,0 +1,18 @@
+feature "user visits show recipe" do
+  before do
+    FactoryBot.create(:recipe, :scrambled_eggs)
+    visit("/")
+  end
+
+  it "shows user clicked recipe" do
+    page.all('a')[0].click
```

Comment:
1. We don't really know what happens here because code is really vague and general. I think that 
  ```
  page.all(:xpath, '//a[contains(text(), "Show")]')[1].click #  1 represents second recipe since indices start from 0
  ```

  will tell a little bit more to non familiar people.
2. It will start failing as soon as you add more links to the page.
3. This is part of the preparation therefore please move to the `before` block.

Reply:
I would like to place it in before block but I cannot define any variable to that so I will leave your suggestion as it is but in content block instead.

Reply:
Not sure what do you mean.

> I would like to place it in before block but I cannot define any variable

The xample I provided you does not define any variable.

> so I will leave your suggestion as it is but in content block instead.

It is called **example block**.

Reply:
I have been told I should move it to the before block and so I did but I could not define variable to recall it with click method and ID in an example block so I made it in the same place where old line of code was.

Reply:
Please put here (in comment) example of the code that you tried and error output. Sometimes example explains things better than humanized description.